install/operator/build_local.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# This script will help you to rapidly build and push the operator
```

Comment:
Isn't https://github.com/syndesisio/syndesis/blob/e14a73f690971535a9d009f045e55dae7a2dda73/tools/bin/commands/build#L167 supposed to do that?

Reply:
Kindof ... it doesn't do it locally, it builds the binary in a docker container which takes longer specially for me with a slow internet connection because it has to download all dependencies. I just thought this file could as well be useful for someone else with a slow internet connection and that's why I pushed, but it doesn't really needs to be there